R/modify.R

```diff
@@ -87,6 +89,30 @@ modify.default <- function(.x, .f, ...) {
   .x[] <- map(.x, .f, ...)
   .x
 }
+#' @rdname modify
+#' @export
+modify.integer  <- function (.x, .f, ...) {
```

Comment:
I don’t think it’s necessary to document these individual methods - I think it makes the docs harder to understand.